src/main/webapp/resources/css/opd_five_five.css

```
@@ -22,6 +22,7 @@
         font-size: 9px!important;
         font-size: 80%;
         text-transform: capitalize!important;
+        page-break-before: always!important;
```

Stylelint (17.9.0)

[error] Expected "page-break-before" to be "break-before" (property-no-deprecated)

(property-no-deprecated)